libbuteosyncfw/common/TransportTracker.h

```diff
@@ -97,7 +97,7 @@ private slots:
     void onUsbStateChanged(bool aConnected);
 
 #ifdef HAVE_BLUEZ_5
-    void onBtStateChanged(QString interface, QVariantMap changed, QStringList invalidated);
+    void onBtStateChanged(const QString interface, const QVariantMap changed, const QStringList invalidated);
```

Comment:
If you change the signatures, could you make these const references instead of plain consts. That's how this kind of data is commonly passed and const on parameters is halfway pointless anyway as the caller shouldn't care if the callee adjusts an internal copy.

Reply:
you are right - did in 02b2b69